prodsys/simulation/control.py

```diff
@@ -510,9 +570,29 @@ def run_process(
             self.env.now,
             state.StateTypeEnum.transport,
         )
-        input_state.process = self.env.process(
-            input_state.process_state(target=target_location)  # type: ignore False
+        flag = False
```

Comment:
make this more elegant, by reusing the same elementary transport function with both normal and link transport